internal/apply/apply.go

```diff
@@ -41,21 +50,26 @@ Only a single environment may be suitable for an initial test, but for a real sy
 
 	flog.Infof("Infrastructure executor: %s", "Terraform")
 
-	applyAll(rootDir, *projectConfig, environments)
+	err = modulesWalkCmd("apply", rootDir, projectConfig, []string{"make"}, environments)
```

Comment:
Will this stop on the first one that fails? We should really test all the modules, and then return the state / output of all the tests. It wouldn't be a great experience to have to run the app multiple times, installing stuff between runs.

Reply:
yeah good point, right now it fails on the first error